applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -627,6 +628,7 @@ void KratosStructuralMechanicsApplication::Register() {
     KRATOS_REGISTER_ELEMENT("AxisymUpdatedLagrangianElement2D9N", mAxisymUpdatedLagrangian2D9N)
 
     // Register the spring damper element
+    KRATOS_REGISTER_ELEMENT("SpringDamperElement2D2N", mSpringDamperElement2D2N);
```

Comment:
```suggestion
    KRATOS_REGISTER_ELEMENT("SpringDamperElement2D", mSpringDamperElement2D);
    KRATOS_REGISTER_ELEMENT("SpringDamperElement3D", mSpringDamperElement3D);
```

how about simplifying this?